librocksdb_sys/build.rs

```diff
@@ -13,7 +13,15 @@ macro_rules! t {
 }
 
 fn main() {
+    const rocksdb_version: &'static str = "5.4";
```

Comment:
Many thanks.
According to rust style guide, statics and constants use SCREAMING_SNAKE_CASE.
better make it `ROCKSDB_VERSION`

😀